parachains/runtimes/bridge-hubs/bridge-hub-polkadot/src/xcm_config.rs

```diff
@@ -197,7 +199,7 @@ impl xcm_executor::Config for XcmConfig {
 		MaxInstructions,
 	>;
 	type Trader =
-		UsingComponents<WeightToFee, DotRelayLocation, AccountId, Balances, ToAuthor<Runtime>>;
+		UsingComponents<WeightToFee, DotRelayLocation, AccountId, Balances, ToStakingPot<Runtime>>;
```

Comment:
@joepetrowski 
I am not sure here, why others (assets, collectives) have `ToStakingPot` and here we have `ToAuthor`,

can you, please, confirm this change?
`ToAuthor`->`ToStakingPot`  and upper `CurrencyAdapter<Balances, ()>` -> `CurrencyAdapter<Balances, DealWithFees<Runtime>>` (DealWithFees uses ToStakingPot)

Reply:
I agree this should go to the staking pot since it is not up to the individual author to process incoming messages.